Apps/WebClient/src/ClientApp/src/store/modules/user/mutations.ts

```diff
@@ -90,6 +91,12 @@ export const mutations: MutationTree<UserState> = {
         state.statusMessage = "success";
         state.status = LoadStatus.LOADED;
     },
+    setPatientData(state: UserState, patientData: PatientData) {
+        state.patientData = patientData;
+        state.error = false;
+        state.statusMessage = "success";
+        state.status = LoadStatus.LOADED;
```

Comment:
this is tricky since having the patient data loaded does not mean that the store is loaded (could be missing user data). Lets add a status of Partial_Loaded to make that difference

Reply:
added PARTIALLY_LOADED status.

Reply:
Is that change updated? Dont see it here yet

Reply:
my bad, mbe I have pushed it after my comment above